examples/nemo/README.md

```diff
@@ -61,7 +98,8 @@ OpenMP parallelisation of traldf_iso over levels.
 ## Example 3
 
 OpenACC parallelisation of tra_adv using the 'data' and 'kernels'
-directives.
+directives. Makes use of the `kernels_trans.py` script from the `scripts`
```

Comment:
Please explain that there is a local script which adds data and kernels then a managed memory script from the scripts directory which only adds kernels and parallel directives.

Reply:
Done.